docs/api.md

```diff
@@ -535,7 +535,7 @@ When `spec.nodeReuse` field of metal3MachineTemplate is set to `True`, CAPM3
 Machine controller:
 
 - Sets `infrastructure.cluster.x-k8s.io/node-reuse` label to the corresponding
-  CAPI object name (`KubeadmControlPlane` or `MachineDeployment`) on the
+  CAPI object name (a `controlplane.cluster.x-k8s.io` object such as `KubeadmControlPlane` or a `MachineDeployment`) on the
```

Comment:
@tmmorin with ^, we are changing the fundamental logic behind the feature, we are not setting `infrastructure.cluster.x-k8s.io/node-reuse` label to the corresponding `type` of owner object on BMH, but the `name` of that object. For example, if CP name is called `cp-test1`, machine controller will set following label on BMH: `infrastructure.cluster.x-k8s.io/node-reuse: cp-test1`